Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/yolo_impl.cpp

```diff
@@ -28,21 +28,21 @@ struct YoloModelImpl : public ModelInterface {
 	ObjectTracking object_tracking;
 	/// <summary>
 	/// Constrcutor to create a arknet-based object recognition network
+	/// Default resolution is set to 640x480
 	/// </summary>
 	/// <param name="modelConfiguration">relative path to the cfg files where the structures of network are defined</param>
 	/// <param name="modelWeights">relative path to the weight files where weights for correspond cfg-file are defined</param>
 	/// <param name="class_names_path">relative path to the txt file where classes of objects are defined </param>
 	YoloModelImpl(string modelConfiguration, string modelWeights, const string class_names_path)
 	{
 		cout << "Constructing a darknet-based model impl\n";
-		
 		net = cv::dnn::readNetFromDarknet(modelConfiguration,modelWeights);
-		
 		net.setPreferableBackend(cv::dnn::DNN_BACKEND_OPENCV);
 		// Use configuration to speed up the net.forward
 		net.setPreferableTarget(cv::dnn::DNN_TARGET_OPENCL);		
-		
 		class_names = read_class_name_file(class_names_path);
+		inWidth = 640;
+		inHeight = 480;
```

Comment:
Again perhaps these variables could be in the super constructor. Not 100% needed though.